app/views/catalog/_index_header_list_default.html.erb

```diff
@@ -1,6 +1,9 @@
 <%# OVERRIDE Hyrax v5.0.1 to collection badge and markdown of links %>
 <div class="search-results-title-row col-12 d-flex flex-row align-items-center pb-2">
-  <h4 class="search-result-title"><%= link_to markdown(document.title_or_label), generate_work_url(document, request) %></h4>
+  <h4 class="search-result-title">
+    <%= link_to document.title_or_label, generate_work_url(document, request, params) %>
```

Comment:
Isn’t markdown an added feature? Why are we removing it?

Reply:
I think this was an accidental change between several revisions of this line. It should definitely go back. Good catch.